test/e2e/kafka_source_test.go

```diff
@@ -1,5 +1,5 @@
-//+build e2e mtsource
-//+build source
+// +build e2e
+// +build source mtsource
```

Comment:
I think this is more correct. 

It means: `e2e AND (source OR mtsource)`. 

Previous have meant: `(e2e or mtsource) AND source`, which is weird.